conf/messages

```diff
@@ -469,6 +470,7 @@ fh.declaration.person_status.label=Job title
 fh.declaration.jobTitle.error.required=Enter the job title of who is submitting this application
 fh.declaration.person_status.error.required=Enter the job title of who is submitting this application
 fh.declaration.person_status.error.pattern=Enter only valid characters for a job title
+fh.declaration.jobTitle.error.pattern=Enter only valid characters for a job title
```

Comment:
is this replacing the one above?  If not, why would 2 be needed?

Reply:
Yes, it is replacing the one above. As of now I am not deleting the previous one incase it break anything else. When all the changes are done , we can do final cleanup of duplicate messages